docs/recipes/Enabling-Serverside-Rendering.md

```diff
@@ -2,8 +2,103 @@
 [👈 Return to Overview](./README.md)
 <!--ONE-DOCS-HIDE end-->
 
-# Enabling Serverside Rendering
+# Enabling Server-side Rendering
 
-Recipe is forthcoming.
+One App will always attempt render your module on the server before sending html back to the client. However, it won't wait for any asynchronous tasks, such as data fetching, to complete before doing so.
+
+The following options provide ways to ensure that server side requested data is included as part of the initial render.
+
+## Using `loadModuleData`
+
+One App v5 introduced [`loadModuleData`](https://one-amex-docs.americanexpress.com/en-us/one-app/api/modules/loading-data) which is invoked on the server before rendering a module.
```

Comment:
might be worth mentioning that `loadModuleData` runs on the server AND client again here? I see the comment below but might be important to clarify since it could cause confusion